onadata/settings/base.py

```diff
@@ -321,7 +321,7 @@ def skip_suspicious_operations(record):
 SECURE_HSTS_SECONDS = env.int('SECURE_HSTS_SECONDS', 0)
 
 # Limit sessions to 1 week (the default is 2 weeks)
-SESSION_COOKIE_AGE = 604800
+SESSION_COOKIE_AGE = env.int('KC_DJANGO_SESSION_LENGTH_HOURS', 168) * 60 * 60
```

Comment:
See https://github.com/kobotoolbox/kpi/pull/4238#discussion_r1101528076 and https://github.com/kobotoolbox/kobo-install/pull/215/files#r1101556640

Reply:
As discussed [here](https://github.com/kobotoolbox/kobo-install/pull/215#discussion_r1101607071), I would modify the PR title and description.